docs/howto/features/cryptnono.md

```diff
@@ -0,0 +1,176 @@
+(howto:features:cryptnono)=
+# Enable stronger anti-crypto abuse features for a hub
+
+For JupyterHubs and BinderHubs that are broadly open to the public, cryptomining
+attacks are the most common security threat. They take up resources and rack up
+cloud costs. While most hubs can get away with gating access to them via some
+kind of login restriction, for a subset of hubs this is not ideal (for equity and
+access reasons).
+
+For those cases, we enable a stronger deployment of [cryptnono](https://github.com/cryptnono/cryptnono).
+The cryptnono project (in use on mybinder.org as well) helps detect and kill cryptominers via
+various 'Detectors'.
+
+## Detectors
+
+Cryptnono currently has two primary detectors:
+
+1. A detector for the [monero](https://www.getmonero.org/) cryptocurrency, that is based on
+   official guidance [from the monero project](https://blog.px.dev/detect-monero-miners/) on
+   how to detect it. This is fairly safe and has a very low false positive rate, and requires
+   no configuration. So by default, **this detector is enabled on all hubs**.
+
+2. A detector (`execWhacker`) based on heuristics, where the full commandline used to execute a process (
```

Comment:
This is capitalized as `execWhacker` but the chart accept `execwhacker` rather than `execWhacker`.

I think it would be good to consistently refer to one capitalization found in the chart config, or something clearly distinguished as written for human eyes, like `the exec whacker`.

Reply:
Yes, I'll do that. I think when writing cryptnono I initially had called it `execWhacker` before moving to all lowercase - I'll think about what's the most appropriate casing, and unify that (including possibly changing it in cryptnono itself).

Reply:
I've just moved everything to say `execwhacker` for now, I'll update it here if I change it in cryptnono